weblate/templates/list-translations.html

```diff
@@ -65,6 +67,9 @@
 {{ suggestions }}
 {% endif %}
 </td>
+<td class="number col-comments">
+{{ trans.stats.comments }}
```

Comment:
Can you please make this link similarly like with other categories? The link target will be same, just with `type=comments`.

Reply:
You mean like having reference as with the line 63?